docs/user/tutorials/01-50-expose-and-secure-a-workload/v2alpha1/01-53-expose-and-secure-workload-ext-auth.md

```diff
@@ -5,85 +5,68 @@ Learn how to expose and secure services using APIGateway Controller and OAuth2.0
 
 ## Prerequisites
 
-* [Deploy a sample HTTPBin Service](../../01-00-create-workload.md).
-* [Set up your custom domain](../../01-10-setup-custom-domain-for-workload.md) or use a Kyma domain instead.
-* [Obtain a JSON Web Token (JWT)](../01-51-get-jwt.md).
+* You have a deployed workload.
+* To use CLI instructions, you must install [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) and [curl](https://curl.se/). Alternatively, you can use Kyma dashboard.
```

Comment:
There's no instruction for Kyma dashboard in the doc, so maybe it shouldn't be mentioned. Also, the heading in line 24 seems redundant.